src/conteiners/App.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import Header from '../components/Header';
+import './App.css';
+import CardList from '../components/CardList';
```

Comment:
Please follow the same naming/reference nomenclature. imports as well as component/folder names are case-sensitive here. Still you're referring to them in a lower-case while having folder names in an uppercase (Components):
![image](https://user-images.githubusercontent.com/52723192/108278098-8faffb80-718b-11eb-8db6-201587a31b95.png)

And below you have folder `conteiners` in a lower-case.